### PR TITLE
Updated link to documentation in the uPortal Links portlet

### PR DESCRIPTION
**Pull request by jonathanmtran**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

Updated link to documentation in the uPortal Links portlet

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: data/quickstart/portlet-definition/uportal-links.portlet-definition.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
                 &lt;h2&gt;uPortal Links&lt;/h2&gt;
                 &lt;ul&gt;
                     &lt;li&gt;&lt;a target=&quot;_blank&quot; href=&quot;http://www.apereo.org/uportal&quot;&gt;uPortal Home&lt;/a&gt;&lt;/li&gt;
-                    &lt;li&gt;&lt;a target=&quot;_blank&quot; href=&quot;http://wiki.jasig.org/display/UPM40/Home&quot;&gt;uPortal 4 Manual&lt;/a&gt;&lt;/li&gt;
+                    &lt;li&gt;&lt;a target=&quot;_blank&quot; href=&quot;https://github.com/Jasig/uPortal/tree/master/docs&quot;&gt;uPortal 5 Documentation&lt;/a&gt;&lt;/li&gt;
```
Comment (ChristianMurphy) [inline data/quickstart/portlet-definition/uportal-links.portlet-definition.xml]: :link: https://github.com/Jasig/uPortal/tree/master/docs :arrow_right: https://jasig.github.io/uPortal/

**Comment by drewwills** [comment]
You rock @jonathanmtran!

**Comment by bjagg** [comment]
:+1:

File: data/quickstart/portlet-definition/uportal-links.portlet-definition.xml (hunk 1 above)
Comment (jonathanmtran) [reply data/quickstart/portlet-definition/uportal-links.portlet-definition.xml]: Noted. Update coming soon.

**Pull request merged**